app/domain/time_entry/description_rules.rb

```diff
@@ -1,6 +1,7 @@
 class TimeEntry::DescriptionRules
   JIRA_REGEX = /(?:\s|^)([A-Z]+-[0-9]+)(?=\s|$)/
   MIN_WORD_COUNT = 4
+  KEYWORDS = ["#english", "english", "english class", "1:1", "one on one", "1-1", "end of week", "weekly call", "weekly meeting", "emails/slack", "catch up", "catching up", "standup", "stand up", "check-in", "check in", "learning", "learn", "donut"]
```

Comment:
should `email/slack` be separated?

also, I don't know the features of `Regexp.union` in `ruby`, but can we include additional options here such as: `email(s?)`?

Reply:
The reason I included email/slack is because some entries in Noko include that entry so the KEYWORDS is a collection of commonly used words for entries. I can also add more if you have suggestions. I did test out if e(-?)mail(s?) will work and it doesn't

Reply:
oh ok, cool! 

i normally list them all out separately (like my typical morning entry is `"morning stuff: email, slack, jira, github"`).

according to the docs (https://ruby-doc.org/core-2.4.1/Regexp.html#method-c-union) we should be able to include more complex matches if we pass actual `Regexp` objects, instead of strings. but that might be more work than it's worth.

Reply:
Thank you for the suggestions! I will add them to the collection of keywords.

I think for now this using strings is a good solution for this scope. We might need to eventually reconsider how the entries are being validated with more intuitive solutions later down the road so it might not be worth it.